libpromises/crypto.c

```diff
@@ -156,7 +155,7 @@ static void RandomSeed(void)
 
 /* PEM functions need the const cast away, but hopefully the default
  * call-back doesn't actually modify its user-data. */
-static const char priv_passphrase[] = "Cfengine passphrase";
+static const char priv_passphrase[] = PRIVKEY_PASSPHRASE;
```

Comment:
I'm not really sure. But if the PEM functions don't take a ``const char*`` I think it's safer to have this array. But in general I was just trying to avoid issues in the change I needed.